packages/runtime/src/helpers/files.ts

```diff
@@ -85,7 +85,7 @@ export const moveStaticPages = async ({
   basePath?: string
 }): Promise<void> => {
   console.log('Moving static page files to serve from CDN...')
-  const outputDir = join(netlifyConfig.build.publish, target === 'server' ? 'server' : 'serverless')
+  const outputDir = join(netlifyConfig.build.publish, target === 'serverless' ? 'serverless' : 'server')
```

Comment:
Unless folks are on the latest Next.js, we'll need to support the new and old naming I believe?

Reply:
The default value of `target` on the latest Next.js is `undefined` and on the previous version it's `'server'`, which both result in `'server'`. So I think it's okay, unless I'm misunderstanding?